src/lsp_data.rs

```diff
@@ -80,6 +87,16 @@ pub mod ls_util {
         }
     }
 
+    pub fn location_from_racer_match(mtch: racer::Match) -> Option<Location> {
+        let source_path = &mtch.filepath;
+
+        if let Some(racer::Coordinate { line, column }) = mtch.coords {
+            Some(location_from_position(source_path, line - 1, column))
+        } else {
+            None
+        }
+    }
+
```

Comment:
Sounds good. Is this project maintaining the ls_types crate? Might be nice to add a racer feature there for a `impl Into<racer::Location> for ls_types::Position` like we did with the `impl racer::FileLoader for rls_vfs::Vfs`.